src/SFA.DAS.EAS.Portal.UnitTests/Portal/Application/EventHandlerTestsFixture.cs

```diff
@@ -0,0 +1,31 @@
+﻿using AutoFixture;
+using SFA.DAS.EAS.Portal.Application.EventHandlers;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EAS.Portal.UnitTests.Portal.Application
+{
+    public class EventHandlerTestsFixture<TEvent, TEventHandler>
```

Comment:
All the common code in the event handler test fixture base class has been duplicated by copy and pasting into each event handler's test fixture. We should move the common code back into the common test fixture. Duplicating the code makes maintenance more time consuming.